src/msspreadsheets/Module.md

```diff
@@ -0,0 +1,23 @@
+Connects to a Microsoft Office 365 Workbook located on Microsoft OneDrive
+
+# Module Overview
+This module contains operations to perform CRUD (Create, Read, Update, and Delete) operations on [Excel workbooks](https://docs.microsoft.com/en-us/graph/api/resources/excel?view=graph-rest-1.0) stored in Microsoft OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | 1.2.0   |
+| Microsoftgraph REST API | v1.0          |
+
+## Supported Operations
+- Open workbook
+- Create worksheet
+- Open worksheet
+- Remove worksheet
+- Create table
+- Rename table
+- Set Table header
+- Insert data into table
+
+## Sample Application
+Please refer the sample application on the README.
```

Comment:
```suggestion
Refer the sample application on the README.
```
Shall we link this to the "README"?

Reply:
Based on the comment from Danesh, I will add some code snippets of supported API calls here. Hence I will remove the highlighted sentence.